src/session.ts

```diff
@@ -165,19 +166,23 @@ export class SessionManager implements Middleware {
                 }
             }
 
+            // Generate a random id for the named pipes in case they have multiple instances of PSES running
+            const id = crypto.randomBytes(10).toString("hex");
             this.editorServicesArgs =
-                "-HostName 'Visual Studio Code Host' " +
-                "-HostProfileId 'Microsoft.VSCode' " +
-                "-HostVersion '" + this.hostVersion + "' " +
-                "-AdditionalModules @('PowerShellEditorServices.VSCode') " +
-                "-BundledModulesPath '" + this.bundledModulesPath + "' " +
-                "-EnableConsoleRepl ";
+                `-HostName 'Visual Studio Code Host' ` +
```

Comment:
Thoughts on using ordinary strings where no interpolation is needed vs consistency? Not opinionated either way.

Reply:
for here, I just kept it consistent with the rest of the string concat